RouteToDo/BackendRepository.h

```diff
@@ -33,6 +33,7 @@
 #pragma mark - Actions
 
 - (void)toggleRouteFavoriteWithUser:(User *)user route:(Route *)route completion:(void (^)(NSError *error))completion;
+- (void)untoggleRouteFavoriteWithUser:(User *)user route:(Route *)route completion:(void (^)(NSError *error))completion;
```

Comment:
I don't think this is necessary. Toggle means its going to flip the value to true or false